### PR TITLE
React week3 hiren

### PR DESCRIPTION
**Pull request by hirenkumar91**
Dear Mentor here is My Homework for week 3

FireShot Capture 002 - Galactica - zany-space-telegram-w64wrr4p5wv35wpw-3000 app github dev
FireShot Capture 003 - Galactica - zany-space-telegram-w64wrr4p5wv35wpw-3000 app github dev

### CODE REVIEW

File: app/about_us/aboutCSS/crew.module.css
Comment (vdaguenet) [inline app/about_us/aboutCSS/crew.module.css:1]: Mind the indentation in this file

File: app/about_us/aboutCSS/partners.module.css
Comment (vdaguenet) [inline app/about_us/aboutCSS/partners.module.css:1]: Indentation

File: app/about_us/ourCrew.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+        imageUrl: "./crew/image-anousheh-ansari.png",
+        description:
+          "Maya brings a unique blend of technical skills and customer service experience to the team. She’s always ready to assist with any needs and to make sure every traveler has an unforgettable experience.",
+      },
+];
+
+const CrewDetails =({ name, title,imageUrl,description}) =>{
+    return(
+    <div className={styles.crewCard}>
+    <div class={styles.cardImg}>
```
Comment (vdaguenet) [inline app/about_us/ourCrew.js]: Don't forget to indent the JSX part as normal HTML

**Review by vdaguenet — APPROVED** [review]
Your implementation is fine but don't forget to indent your code properly.

File: app/about_us/aboutCSS/crew.module.css
Comment (hirenkumar91) [reply app/about_us/aboutCSS/crew.module.css:1]: Indentation done:

File: app/about_us/aboutCSS/partners.module.css
Comment (hirenkumar91) [reply app/about_us/aboutCSS/partners.module.css:1]: Indentation done:

File: app/about_us/ourCrew.js (hunk 1 above)
Comment (hirenkumar91) [reply app/about_us/ourCrew.js]: Indentation done:

**Comment by hirenkumar91** [comment]
Dear Mentor ; all code indentation done , should i merge branch

**Review by vdaguenet — APPROVED** [review]

**Pull request merged**